src/tests/ftest/object/PunchTest.py

```diff
@@ -27,12 +27,9 @@
 import sys
 import json
 import avocado
-from avocado import Test, main
+from apricot import Test
+from avocado import main
```

Comment:
(pylint-unused-import) Unused main imported from avocado

(pylint-ungrouped-imports) Imports from package avocado are not grouped